apps/coscrad-frontend/src/components/resources/vocabulary-lists/vocabulary-list-detail.thumbnail.presenter.tsx

```diff
@@ -0,0 +1,24 @@
+import {
+    IDetailQueryResult,
+    IVocabularyListViewModel,
+    ResourceType,
+} from '@coscrad/api-interfaces';
+import { Card, CardContent, Divider } from '@mui/material';
+import { Link } from 'react-router-dom';
+import { routes } from '../../../app/routes/routes';
+import { formatBilingualText } from './utils';
+
+export const VocabularyListDetailThumbnailPresenter = ({
+    data: { id, name, nameEnglish, entries },
+}: IDetailQueryResult<IVocabularyListViewModel>): JSX.Element => (
+    <Link to={`/${routes.resources.ofType(ResourceType.vocabularyList).detail(id)}`}>
+        <Card>
+            <CardContent>
+                <h3 data-testid={id}>{formatBilingualText(name, nameEnglish)}</h3>
```

Comment:
Could use MUI CardHeader component for Title and MUI CardContent for the card info.

Reply:
Done.